spec/poltergeist_helper.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
```

Comment:
My thinking at the time was that it had very limited scope and that we might kill all those specs, entirely with the introduction of the smoke tests.